dumpling/export/consistency.go

```diff
@@ -116,6 +118,15 @@ type ConsistencyLockDumpingTables struct {
 
 // Setup implements ConsistencyController.Setup
 func (c *ConsistencyLockDumpingTables) Setup(tctx *tcontext.Context) error {
+	if c.conf.ServerInfo.ServerType == ServerTypeTiDB {
+		if enableTableLock, err := CheckTiDBEnableTableLock(c.conn); err != nil || !enableTableLock {
```

Comment:
if `CheckTiDBEnableTableLock` returns error, user can't continue dumping 🤔 Can we only check `CheckTiDBEnableTableLock` when needed?

Reply:
This check will only run when we dump tidb tables with consistency lock. Because without `enable-table-lock` TiDB will return a successfully result from `LOCK TABLES` but actually it doesn't have any effect, so we need to make sure this variable is set before we lock tables.

Reply:
Oh I noticed the receiver is `ConsistencyLockDumpingTables`